docs/release.sh

```diff
@@ -78,6 +78,11 @@ upload_current_documentation() {
 	src=site/
 	dst=s3://$BUCKET$1
 
+	cache="max-age=3600"
+	if [ ! -z "$NOCACHE" ]; then
```

Comment:
Also, you have some unnecessary interpolation in this same file's changes that I'll leave as an exercise for you to figure out where is at. :wink: